backend/Models/Seat.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace CinemaApp.Models
+{
+    public class Seat : IEquatable<Seat>
```

Comment:
as this is a Data Model in the database (a different table) please add an Id to it.